mender-update/deployments.cpp

```diff
@@ -222,6 +222,94 @@ error::Error CheckNewDeployments(
 	return client.AsyncCall(v2_req, header_handler, v2_body_handler);
 }
 
+static const string
+	deployment_status_strings[static_cast<int>(DeploymentStatus::AlreadyInstalled) + 1] = {
```

Comment:
Suggest adding a `DeploymentStatus::End` to the list and using that instead (without `+ 1`). It's a bit more future proof than using one of the elements.